qa/integration/specs/multiple_pipeline_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_relative '../framework/fixture'
+require_relative '../framework/settings'
+require_relative '../services/logstash_service'
+require_relative '../framework/helpers'
+require "logstash/devutils/rspec/spec_helper"
+require "socket"
+require "yaml"
+
+describe "Test Logstash service when multiple pipelines are used" do
+  before(:all) {
+    @fixture = Fixture.new(__FILE__)
+  }
+
+  after(:all) {
+    @fixture.teardown
+  }
+
+  let(:pipelines) {[
+    {
+      "pipeline.id" => "test",
+      "pipeline.workers" => 1,
+      "pipeline.batch.size" => 1,
+      "config.string" => "input { } output { }"
```

Comment:
Can we use the configs `generator { count => 1 } output { file => "blah1"}` and `generator { count => 1 } output { file => "blah2"}`.

We can then check if blah1 and blah2 files are non empty. That's more comprehensive, right?